docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -512,7 +512,7 @@ Follow these steps:
 
 ## Proposing a Pull Request (PR)
 
-After you've committed your changes, check here for [how to open a Pull Request](/how-to-open-a-pull-request.md).
+After you've committed your changes, check here for [how to open a Pull Request](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-open-a-pull-request.md).
```

Comment:
```suggestion
After you've committed your changes, check here for [how to open a Pull Request](how-to-open-a-pull-request.md).
```
Absolute links work on Github, but do not work properly on contribute.freecodecamp.org, so we need to use relative links (which work on both).